decisions/wc_undead_decisions.txt

```diff
@@ -148,6 +148,7 @@ decisions = {
 				trait = creature_plague_eruptor
 				trait = creature_abomination
 				trait = creature_lich
+				trait = creature_wicker_golem
```

Comment:
Reminder: `creature_wicker_golem` needs buffing since `creature_wicker_behemoth` is better.

Reply:
changed stats, pls verify whether these are desirable